src/index.js

```diff
@@ -1,17 +1,33 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import App from './components/App';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import allReducers from './reducers/index';
+
+export const initState = {
+  books: [
+    {
+      author: 'JK Rowlings',
+      id: Math.random(),
```

Comment:
**[OPTIONAL]**

Math.random does not guarantee unique number generation, we can use libraries like `short-id` or `uuid`